license/src/com/untangle/app/license/LicenseManagerImpl.java

```diff
@@ -219,7 +225,7 @@ public void run()
     @Override
     public final License getLicense(String identifier, boolean exactMatch)
     {
-        if (isGPLApp(identifier))
+        if (!isRestricted() && isGPLApp(identifier))
```

Comment:
confirm if license service is returning GPL apps in license response, so that this can be removed

Reply:
The free apps are in the getStatus of the license API, but they are not returned in the response.